evap/evaluation/models.py

```diff
@@ -2017,3 +2017,10 @@ def send_textanswer_reminder_to_user(cls, user: UserProfile, evaluation_url_tupl
         body_params = {"user": user, "evaluation_url_tuples": evaluation_url_tuples}
         template = cls.objects.get(name=cls.TEXT_ANSWER_REVIEW_REMINDER)
         template.send_to_user(user, {}, body_params, use_cc=False)
+
+
+class VoteTimestamp(models.Model):
+    """A timestamp, when a user voted for a course."""
+
+    evaluation = models.ForeignKey(Evaluation, models.PROTECT)
```

Comment:
I don't think we want `models.PROTECT` here -- this is basically a kind of log entry, if an evaluation is deleted, we should just also delete all timestamps stored for that evaluation.